src/models/state.js

```diff
@@ -0,0 +1,35 @@
+import Immutable from 'seamless-immutable';
```

Comment:
dont support this idea, i think in the short future we supposed to implement settings so that you can edit them easly from ui, i mean scene and redux stuff